File: IT/1/Strutture dati/Esercizi/C++/esercizio3.cpp
/* file esercizio3.cpp */

#include <iostream>

using std::cout;
using std::endl;

int main()
{
	int nums[] = {1, 3, 5, 7, 9};

	int i = sizeof(nums) / sizeof(int) - 1;

	while (i >= 0) {
		cout << nums[i] << " ";
		i--;
	}

	cout << endl;
}

File: IT/1/Strutture dati/Esercizi/Java/Esercizio3.java
/* file Esercizio3.java */

public class Esercizio3 {
	public static void main(String [] args) {
		int [] nums = {1, 3, 5, 7, 9};

		int i = nums.length - 1;

		while (i >= 0) {
			System.out.print(nums[i] + " ");
			i--;
		}

		System.out.print(System.lineSeparator());
	}
}

